datasets/speechocean762/README.md

```diff
@@ -0,0 +1,260 @@
+
+-`pronunciation scoring`: This dataset is designed for pronunciation assessment at sentence-level, word-level and phoneme-level.
+
+There is a [leaderboard](https://paperswithcode.com/dataset/speechocean762) for phoneme-level scoring on Papers With Code.
+
+
+
+### Languages
+
+English. Note that the mother tongue of all the speakers is Mandarin. 
+
+## Dataset Structure
+
+### Data Instances
+
+Below is a typical meta data of an instance of the dataset:
+
+```
+"014350017": {    ,
+    "speaker": "1435",
+    "gender": "m",
+    "age": 7,
+    "text": "STEEVEN IS GOING TO SEE WHALE",
+    "accuracy": 7,
+    "completeness": 10,
+    "fluency": 7,
+    "prosodic": 7,
+    "total": 6,
+    "words": [
+        {
+        "accuracy": 10,
+        "phones": ["S", "T", "IY1", "V", "AH0", "N"],
+        "phones-accuracy": [2, 2, 2, 1.6, 2, 2],
+        "stress": 10,
+        "text": "STEEVEN",
+        "total": 10,
+        "mispronunciations": []
+        },
+        {
+        "accuracy": 10,
+        "phones": ["IH0", "Z"],
+        "phones-accuracy": [2, 2],
+        "stress": 10,
+        "text": "IS",
+        "total": 10,
+        "mispronunciations": []
+        },
+        {
+        "accuracy": 10,
+        "phones": ["G", "OW0", "IH0", "NG"],
+        "phones-accuracy": [
+            2, 2, 2, 2],
+        "stress": 10,
+        "text": "GOING",
+        "total": 10,
+        "mispronunciations": []
+        },
+        {
+        "accuracy": 10,
+        "phones": ["T", "UW0"],
+        "phones-accuracy": [2, 1.8],
+        "stress": 10,
+        "text": "TO",
+        "total": 10,
+        "mispronunciations": []
+        },
+        {
+        "accuracy": 10,
+        "phones": ["S", "IY0"],
+        "phones-accuracy": [2, 2],
+        "stress": 10,
+        "text": "SEE",
+        "total": 10,
+        "mispronunciations": []
+        },
+        {
+        "accuracy": 2,
+        "mispronunciations": [
+            {
+            "canonical-phone": "W",
+            "index": 0,
+            "pronounced-phone": "<DEL>"
+            },
+            {
+            "canonical-phone": "EY",
+            "index": 1,
+            "pronounced-phone": "OW"
+            },
+            {
+            "canonical-phone": "L",
+            "index": 2,
+            "pronounced-phone": "L*"
+            }
+       ],
+        "phones": ["W", "EY0", "L"],
+        "phones-accuracy": [0, 0, 0.4],
+        "stress": 10,
+        "text": "WHALE",
+        "total": 3
+        }
+   ]
+}
+```
+
+### Data Fields
+
+- text: transcript text
+
+- accuracy: the pronunciation accuracy
+
+- completeness: the percentage of the words in the target text that are actually pronounced
+
+- fluency: intonation, stable speaking speed and rhythm
+
+- stress: whether the stress position is correct
+
+- mispronunciation: only available when phone accuracy is lower than 0.5. Indicates which phoneme the current phone was actually pronounced.
+
```

Comment:
Can you also describe all the other fields here please ? Especially the "words" one if possible